### ISSUE TITLE
window.showModalDialog("CreateDirectoryAction", obj)  console output only when the dialog is moved

### ISSUE THREAD

**Issue by Ning2018** [Ning2018/Bugs#7]
in main.jsp

	function showCreateDirDialog() {

		var obj = new Object();
	//	obj.path = getPath();
	//	obj.window = window;

		//   window.showModalDialog("CreateDirectoryAction", obj, 20, 11);

		window.showModalDialog("CreateDirectoryAction", obj);
<%-- show_modal_dialog("create_dir_page.action", obj, 20, 11);  --%>
	}

in struts.xml

		<action name="CreateDirectoryAction"  class="CreateDirectoryAction">
		<result name="success">/WEB-INF/jsp/create_directory.jsp</result>
		<result name="error">/WEB-INF/jsp/errorpage.jsp</result>
		</action>

in create_directory.jsp

<%@ page language="java" contentType="text/html; charset=UTF-8"
	pageEncoding="UTF-8"%>
<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Build Directory</title>
<script src="../javascript/prototype.js" type="text/javascript"></script>
<script type="text/javascript" src="../javascript/common.js"></script>
<link type="text/css" rel="stylesheet" href="css/style.css" />

</head>

<body>

	<c:choose>
		<c:when test="${param.currentPath==null}">
			<input type="hidden" value="/" name="txt_path" />
		</c:when>
		<c:otherwise>
			<input type="hidden" value="${param.currentPath}" name="txt_path" />
		</c:otherwise>
	</c:choose>

	<div id="div" style="left: 10px; top: 10px; position: absolute;">
		请输入文件夹名称： <input type="text" name="fileName" /> <input type="hidden"
			name="parentPath" id="parentPath" />
		<p />
		当前路径：<label id="lbl_path" value="${param.currentPath}" ></label>
		<p />
		<input type="button" value="确定" onclick="jsonCreateDir()" />

	</div>

	<script type="text/javascript">
		function getPath() {
			return $("txt_path").value;
		}
		function setPath(path) {
			$("txt_path").value = path;

			showCurrentPath();
		}
		function showCurrentPath() {
			var divCurrentPath = document.getElementById("div_current_path");

			if (divCurrentPath) {

				divCurrentPath.innerHTML = "\u5f53\u524d\u8def\u5f84\uff1a"
						+ getPath();

			}
		}
		function jsonCreateDir() {
			var url = 'createDirectoryAction';

			var params = Form.serialize('div');

			var myAjax = new Ajax.Request(url, {

				method : 'post',

				parameters : params,

				onComplete : processResponse,

				asynchronous : true
			});
		}
		function processResponse(request) {

			var obj = request.responseText.evalJSON();
			//alert(obj.result);
			if (obj.name == undefined)
				return;
			addRowToTable(parentWindow, "tbl_list", new Array(packageDir(obj),
					obj.time, getFormatSize(obj.size)));

			window.close();
		}

		function addRowToTable(mywindow, id, values) {

			var table = mywindow.document.getElementById(id);
			if (table) {
				tr = table.insertRow();

				tr.style.height = " 20px";
				for (var i = 0; i < values.length; i++) {
					td = tr.insertCell();
					td.innerHTML = values[i];
					td.id = i;
					if (i == values.length - 1)
						td.style.textAlign = "right";
				}
				return tr;
			}
		}
	</script>
	<script type="text/javascript">
		var path = window.dialogArguments.path;
		var parentWindow = window.dialogArguments.window;
		;
		var lbl_path = document.getElementById("lbl_path");
		var parentPath = document.getElementById("parentPath");
		lbl_path.innerHTML = path;
		parentPath.value = path;
	</script>

</body>
</html>

**Comment by Ning2018** [comment]
currentPath: undefined

**Comment by Ning2018** [comment]
.js file

cache
